### DISCUSSION TITLE
Mandibular reconstruction surgery made with Slicer’s BoneReconstructionPlanner extension

### DISCUSSION THREAD

**Discussion by qferreira-1** [lassoan/SlicerBoneReconstructionPlanner#100] [General]
We have performed a mandibular reconstruction with a fibular free flap at Sinai Health using 3D Slicer's bone reconstruction planner. My apologies for the long wait to post; unfortunately, I won't be able to post intra-op photos due to consent. This was the first time Sinai had used in-house guides, and it was a success with both fitment of the guide onto the mandible and fibula, as well as placement of the osteotomies into the pre-bent plate. 
There was an SCC on the left body mandible where the condylar process and leftmost lateral incisor would need to be removed. The first image is the model of the patient's tumor; the alveolar nerve was damaged so further resection was needed. 
![LeftHead2](https://user-images.githubusercontent.com/122468415/226108646-0da9c559-3937-4109-b122-831ed83bc238.png)

 A 1cm measurement was used from the tumor to decide the cuts. The second image shows the mandible guide setup. 
![MandibleNoCondyle](https://user-images.githubusercontent.com/122468415/226108520-3ab442d7-a49b-488a-ae0f-cf3fe559d3f3.png)

A 7cm measurement from the distal tip of the left fibula was used to mark where the first cut could occur. from here the total length needed was approximated to be 110-120mm and this line was centred in the fibula. The osteotomy lengths were 57.3mm for the distal segment which extends into the condylar process, and 66.21mm which connected the ramus angle to the chin. The guide can be seen in the image below where a 0.6mm blade was being used, so a tolerance of 0.1mm was used. The fixation screws were 2.0-10mm so 2.3 diameter holes were placed. 
![FibGuideNoCondyle](https://user-images.githubusercontent.com/122468415/226109160-92111347-8b76-4190-8c51-594d81ea3593.png)
Here's a photo of it after the print, sorry did not take too many photos, it was a rushed print.
![image](https://user-images.githubusercontent.com/122468415/226109220-8ea109e5-be34-46ac-8443-964054245703.png)
![IMG_4009](https://user-images.githubusercontent.com/122468415/226109240-530e895a-2873-4e62-a6cf-f25d70177f0b.jpg)

This was an SLA printer printed with 0.1mm layer thickness in a biocompatible resin. Minor post-processing was involved.

Overall, everything went great and the program worked very well. This extension takes time to learn, so much practice is needed to understand the extension fully. Excellent work to @mauigna06, @lassoan, and the rest of the team.

**Comment by mauigna06** [comment]
Thanks for the feedback @qferreira-1

Please keep doing these kind of posts

Mauro
